src/Navbar.tsx

```diff
@@ -9,6 +9,9 @@ import NavLink from 'portal-components/NavLink'
 import NavUserLink from './portal-components/NavUserLink'
 import { GenericRoute } from 'types/portal-config'
 import Button from 'react-bootstrap/esm/Button'
+import IconButton from '@material-ui/core/IconButton'
```

Comment:
@yjyoo773 from the issue comments, I thought we decided to convert this to react-bootstrap. Is there a reason you decided to continue using Material-UI?